boilerplate/.apirc

```diff
@@ -0,0 +1,16 @@
+{
+  "persistence": {
+    "postgres": {
+      "connection": {
+        "host": "localhost",
+        "user": "postgres",
+        "password": "postgres",
+        "database": "test_db"
+      },
+      "pool": {
+        "min": "2",
+        "max": "7"
+      }
+    }
+  }
+}
```

Comment:
You have this file in duplicate -- one inside the boilerplate, and one outside, in the root repository dir. Is this necessarily a repository-wide config file, or is it project-specific?

Reply:
so for Docker image we need it inside the repository-folder. but for running it locally we need it in the global scope

Reply:
But then if you make changes to the apirc, you have to remember to make those changes twice, in two different files. There's gotta be a better way to do this.

All services will be running in docker containers right? Is it then necessary to be able to run it locally? If it is, couldn't there be another way to do this without having duplicate files, such as pointing the docker image to the global apirc, or with a symbolic link?

Reply:
by locally I meant using vscode or any tool...because rc loader only reads root folder (atleast on Windows system). I don't know how sym-link would work here

Reply:
I spent some time today fixing this, for future reference -- [this is where rc looks for configuration](https://www.npmjs.com/package/rc#standards).